docs/api-usage/prices.mdx

```diff
@@ -4,6 +4,7 @@ title: Prices
 
 Saleor API has four different types that represent prices. The most basic type consists of a currency code and a numeric amount. The most complex one can define a range of prices, including net, gross, and tax amounts.
 The numeric amount precision is determined by the precision of the price currency. For example, USD has a precision value of 2, such as $1.00.
+Regardless of the currency, the maximum price value that can be provided is 99,999,999,999. If this value is exceeded, an error will be raised.
```

Comment:
`,` is confusing here, what about decimal place? How I understand it will be max digits, including decimal places?

Reply:
I think it depends if you use `,` and decimal places separator or `.` Docs: https://docs.oracle.com/cd/E19455-01/806-0169/overview-9/index.html
I think we can use `99 999 999 999 999`